README.md

```diff
@@ -53,7 +58,9 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+weather = "sunny"; means that weather is assigned a value of "sunny".
+
+weather === "sunny" means that the variable weather evaluates to "sunny"
```

Comment:
Strictly, what this question was asking was the value that each expression evaluated to. 
`weather = "sunny"` not only assigns `weather` the value `"sunny"` - the expression itself also evaluates to `"sunny"`. 
`weather === "sunny"` evaluates to a boolean (in this case, `true`)